tools/sgen/input/FluentUIStyle.yml

```diff
@@ -311,6 +311,16 @@ MSFBorderedCardNudgeTokens extends MSFCardNudgeTokens:
     outlineColor: $Colors.Stroke.neutral1
     textColor: $Colors.Brand.shade20
 
+AP_MSFCommandBarTokens:
```

Comment:
We should change the Demo Controller description to "CommandBar (Vnext)" to communicate that this control is tokenized.